From 9b1a27b4702c517207f1d10fbf71f617ae2a2154 Mon Sep 17 00:00:00 2001
From: Jean-Marie Lemetayer <jean-mariex.lemetayer@intel.com>
Date: Thu, 5 Jul 2012 15:13:21 +0200
Subject: [PORT FROM MAIN] WLAN: Need to remove dependency on .git for generating version number

BZ: 45255

The generation of version.h based on git tag doesn't work well here
because we are building Compat from outside of the git tree.
Moreover, for aosp builds it totally fail because there is no git
tree. So, this patch cancelling the generation of the file and include
the file in the git tree.

Change-Id: Ic4fb8686fd8aaf8a1a253a96e81f7f55167f663b
Orig-Change-Id: I735e841793b18d6b4905e4a3f452e13160b916a9
Signed-off-by: Jean-Marie Lemetayer <jean-mariex.lemetayer@intel.com>
Reviewed-on: http://android.intel.com:8080/55450
Reviewed-by: Trivelly, Jean <jean.trivelly@intel.com>
Reviewed-by: Marotte, Fabien <fabien.marotte@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Signed-off-by: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Reviewed-on: http://android.intel.com:8080/59017
Reviewed-by: Regairaz, Julie <julie.regairaz@intel.com>
Reviewed-on: http://android.intel.com:8080/77968
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/drivers/net/wireless/wl12xx/Makefile |   11 -----------
 .../drivers/net/wireless/wl12xx/version.h          |    2 ++
 2 files changed, 2 insertions(+), 11 deletions(-)
 create mode 100644 wl12xx-compat/drivers/net/wireless/wl12xx/version.h

diff --git a/wl12xx-compat/drivers/net/wireless/wl12xx/Makefile b/wl12xx-compat/drivers/net/wireless/wl12xx/Makefile
index 01b8bb5..b87e8e5 100644
--- a/wl12xx-compat/drivers/net/wireless/wl12xx/Makefile
+++ b/wl12xx-compat/drivers/net/wireless/wl12xx/Makefile
@@ -1,17 +1,6 @@
 wl12xx-objs		= main.o cmd.o io.o event.o tx.o rx.o ps.o acx.o \
 			  boot.o init.o debugfs.o scan.o
 
-define filechk_version.h
-	(echo 'static const char *wl12xx_timestamp = __TIMESTAMP__;'; \
-	echo 'static const char *wl12xx_git_head = \
-			"$(shell git describe --dirty)";')
-endef
-
-$(obj)/version.h: .git/HEAD .git/index .git/refs/tags
-	@$(call filechk,version.h)
-
-$(obj)/main.c: $(src)/version.h
-
 # convert all wl12xx-objs to $(src)/file form
 define WL12XX_OBJS_SRC
 $(shell echo $(wl12xx-objs) |
diff --git a/wl12xx-compat/drivers/net/wireless/wl12xx/version.h b/wl12xx-compat/drivers/net/wireless/wl12xx/version.h
new file mode 100644
index 0000000..114768d
--- /dev/null
+++ b/wl12xx-compat/drivers/net/wireless/wl12xx/version.h
@@ -0,0 +1,2 @@
+static const char *wl12xx_timestamp = __TIMESTAMP__;
+static const char *wl12xx_git_head = "TI_R5.00.18";
-- 
1.7.4.1

